### PR TITLE
Rename Acknowledgment to Resolution

### PR DESCRIPTION
**Pull request by prayerslayer**
Also change API to be truly REST, consistency

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/zalando-stups/fullstop/pull/112" target="_blank">ReviewNinja

File: fullstop-api.yaml [hunk 1]
```diff
@@ -131,28 +136,26 @@ paths:
             items:
               $ref: "#/definitions/Violation"
 
-  /api/violations/{id}:
-    put:
-      summary: Comment and acknowledged violation
-      description: Comment and acknowledged violation
+  /api/resolution/{id}:
```
Comment (mrandi) [inline fullstop-api.yaml]:
```
POST /api/violations/{v_id}/resolution
~~ GET /api/violations/{v_id}/resolution/{id}
```

File: fullstop-api.yaml [hunk 2]
```diff
@@ -131,28 +136,26 @@ paths:
...
         - name: id
           in: path
           type: integer
           required: true
-        - name: acknowledged
+        - name: message
           in: body
           required: true
           schema:
-            $ref: "#/definitions/Acknowledged"
```
Comment (mrandi) [inline fullstop-api.yaml]: I think:

```
type: string
```

**Comment by mrandi** [comment]
close merged in #108

**Pull request closed without merging**